gax-httpjson/src/main/java/com/google/api/gax/httpjson/ApiMessageHttpResponseParser.java

```diff
@@ -91,8 +93,12 @@ public ResponseT parse(InputStream httpResponseBody) {
       return null;
     } else {
       Type responseType = getResponseInstance().getClass();
-      return getResponseMarshaller()
-          .fromJson(new InputStreamReader(httpResponseBody), responseType);
+      try {
+        return getResponseMarshaller()
+            .fromJson(new InputStreamReader(httpResponseBody), responseType);
```

Comment:
Important: this needs a character set, typically detected from the HTTP headers in this case

Reply:
added